metrics/ci_integrations/lib/client/github_actions/mappers/run_conclusion_mapper.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:ci_integration/client/github_actions/models/run_conclusion.dart';
+import 'package:ci_integration/integration/interface/base/client/mapper/mapper.dart';
+
+/// A class that provides methods for mapping conclusions of
+/// Github Actions [WorkflowRun]s.
+class RunConclusionMapper implements Mapper<String, RunConclusion> {
+  /// A conclusion for a successful workflow run.
+  static const String success = 'success';
+
+  /// A conclusion for a failed workflow run.
+  static const String failure = 'failure';
+
+  /// A neutral conclusion for a workflow run.
+  static const String neutral = 'neutral';
+
+  /// A conclusion for a cancelled workflow run.
+  static const String cancelled = 'cancelled';
+
+  /// A conclusion for a skipped workflow run.
+  static const String skipped = 'skipped';
+
+  /// A conclusion for a timed out workflow run.
+  static const String timedOut = 'timed_out';
+
+  /// A conclusion for a workflow run that requires an action.
+  static const String actionRequired = 'action_required';
+
+  /// Creates a new instance of the [RunConclusionMapper].
+  const RunConclusionMapper();
+
+  /// Maps the [conclusion] of a workflow run to the [RunConclusion].
```

Comment:
```suggestion
  /// Maps the given [conclusion] of a workflow run to the [RunConclusion] value.
```

Or maybe we should avoid documenting override methods as we have documentation in the parent class?